src/types/recaptcha.ts

```diff
@@ -0,0 +1,127 @@
+// Type definitions for Google Recaptcha 3.0
+// Copied from: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Action {
+  /**
+   * the name of the action. Actions may only contain alphanumeric characters and slashes, and must not be user-specific.
+   */
+  action: string;
```

Comment:
Can we limit this to a union type? I bet not every string classifies as an action